flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/source/assigner/SplitAssignerTestBase.java

```diff
@@ -101,29 +103,29 @@ private void assertAvailableFuture(
 
   protected void assertGetNext(SplitAssigner assigner, GetSplitResult.Status expectedStatus) {
     GetSplitResult result = assigner.getNext(null);
-    Assert.assertEquals(expectedStatus, result.status());
+    assertThat(result.status()).isEqualTo(expectedStatus);
     switch (expectedStatus) {
       case AVAILABLE:
-        Assert.assertNotNull(result.split());
+        assertThat(result.split()).isNotNull();
         break;
       case CONSTRAINED:
       case UNAVAILABLE:
-        Assert.assertNull(result.split());
+        assertThat(result.split()).isNull();
         break;
       default:
-        Assert.fail("Unknown status: " + expectedStatus);
+        fail(String.format("Unknown status: %s", expectedStatus));
```

Comment:
`String.format` seems redundant because `fail` method supports `(String failureMessage, Object... args)`.

Reply:
Thank for checking this. Fix in the new commit.